controller.rst

```diff
@@ -157,6 +157,23 @@ It supports authentication via various methods including username/password, OAut
 You can also create custom dashboards and alerts to monitor the metrics and logs from the connected units.
 See the `official documentation <https://grafana.com/docs/grafana/latest/>`_ for more information on how to use Grafana.
 
+.. _controller_updating-section:
+
+Unit updates
+============
+
+The controller allows you to update the units directly from the interface, similar to the process in :ref:`the unit web interface<updates-section>`. Two types of updates are available:
+
+- **Package updates**: Update the packages installed on the unit. List and install available updates by clicking on :guilabel:`Check packages updates` in the unit menu.
+  A modal will display the list of available updates. If updates are available, apply them by clicking on the :guilabel:`Update` button in the modal.
```

Comment:
I'll add something like: this is the first thing to try if version awareness blocks you from connecting to the unit.